swift/ql/lib/codeql/swift/frameworks/StandardLibrary/String.qll

```diff
@@ -68,6 +68,10 @@ private class StringSummaries extends SummaryModelCsv {
         ";StringProtocol;true;trimmingCharacters(in:);;;Argument[-1];ReturnValue;taint",
         ";StringProtocol;true;uppercased();;;Argument[-1];ReturnValue;taint",
         ";StringProtocol;true;uppercased(with:);;;Argument[-1];ReturnValue;taint",
+        ";StringProtocol;true;withCString(_:);;;Argument[-1];Argument[0].Parameter[0].CollectionElement;taint",
+        ";StringProtocol;true;withCString(_:);;;Argument[0].ReturnValue;ReturnValue;value",
```

Comment:
Should there be a step from the mutated parameter of the callback to the `this` parameter of the method?

Reply:
Discussed on the other comment.